### PR TITLE
Added docstrings, Typehinting and isinstance for checking the type

### PR DESCRIPTION
**Pull request by Dhravya**

### CODE REVIEW

File: pyrandtoys/functions.py [hunk 1]
```diff
@@ -50,22 +104,42 @@ def switch(count=1):
...
-    if str(type(list)) == "<class 'str'>":
+    Parameters:
+    list (list) : The list of functions to randomise
+
+    Returns:
+    tuple: The values of the randomised functions
+    """
+    res = ()
+    
+    if isinstance(list, list):
```
Comment (thenithinbalaji) [inline pyrandtoys/functions.py:137]: Must be comparing str right?
```
if isinstance(list, str):
        temp = []
        temp.append(list)
        list = temp
```

**Review by thenithinbalaji — CHANGES_REQUESTED** [review]

File: pyrandtoys/functions.py (hunk 1 above)
Comment (Dhravya) [reply pyrandtoys/functions.py:137]: Oh, sorry, my bad

Comment (TrapinchO) [reply pyrandtoys/functions.py]: maybe change it to this?
```python
if isinstance(list, str):
    list = [list]
```

**Comment by Dhravya** [comment]
Hi there, changes have been done, you can merge now

**Review by thenithinbalaji — APPROVED** [review]
LGTM 👍

**Pull request merged**